src/dynamics/joint/WheelJoint.ts

```diff
@@ -584,6 +672,48 @@ export class WheelJoint extends Joint {
       wB += iB * impulse;
     }
 
+    if (this.m_enableLimit) {
```

Comment:
Here is the error: In the car example the impulses in this block get way too large